domino_generator/example/ex2/web-project/ex2.g.dart

```diff
@@ -1,11 +1,14 @@
 import 'package:domino/src/experimental/idom.dart' as _i0 show DomContext;
-import 'components/named-div/button.g.dart' as _i1 show renderButton;
-import 'components/named-div/colorbox.g.dart' as _i2 show renderRedBox;
+import 'package:intl/intl.dart' as _i1 show Intl;
+import 'components/named-div/button.g.dart' as _i2 show renderButton;
+import 'components/named-div/colorbox.g.dart' as _i3 show renderRedBox;
 
 void renderEx2(_i0.DomContext $d) {
   $d.open('button');
-  $d.text('\n        The best button\n    ');
+  String text_10490() => _i1.Intl.message('\n        The best button\n    ',
```

Comment:
At this point it seems that we should normalize this text via trimming it (and maybe replace every whitespace to space).